### DISCUSSION TITLE
!help :I got a BOT GOTV

### DISCUSSION THREAD

**Discussion by CONSTRUCTPRO** [danielkrupinski/Osiris#3873] [Q&A] [answered]
Sometimes I'm getting this spectator watching(he's not playing) ,Is this what you call 'OverWatch'??
![Capture2](https://user-images.githubusercontent.com/98707822/201494634-296701fc-4769-43ca-ac5a-7e1425f703cb.PNG)

**Comment by Xsintashi** [comment, marked as answer]
nop. dont mind, its just a bot for spectators that arent in game

**Reply by CONSTRUCTPRO** [reply]
Many thanks!
